packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -134,7 +134,7 @@ describe('--schema from parent directory', () => {
 function cleanSnapshot(str: string): string {
   return str
     .replace(/\d+ms/g, 'XXms')
-    .replace(/\d+s/g, 'XXs')
+    .replace(/\d+(?>\.\d+)?s/g, 'XXms')
```

Comment:
Do we still need these two replacements here? I think we can just remove them and update the corresponding snapshots to use `XXXms` instead of `XXms`, since both replacements are now handled by the snapshot serializer.

Reply:
I completely removed that code since it's centralized in jestSnapshotSerializer.js